src/components/ui/Shared/Container.tsx

```diff
@@ -0,0 +1,63 @@
+import type { StyleDeclarationValue } from "aphrodite";
+import React from "react";
+import { StyleSheet, css } from "aphrodite";
+
+type ContainerType = "normal" | "fluid";
+
+type Props = {
+  readonly children: React.ReactNode;
```

Comment:
React.ReactNode will basically be a type that will allow this component to be wrapped around other components or text or whatever can go inside a div